.github/workflows/stale.yml

```diff
@@ -17,3 +17,4 @@ jobs:
             stale-pr-label: "stale"
             stale-pr-message: "This pull request is stale because it has been open for 60 days with no activity."
             close-pr-message: "This pull request was closed because it has been inactive for 7 days since being marked as stale."
+            operations-per-run: 300
```

Comment:
From https://github.com/marketplace/actions/close-stale-issues#operations-per-run

>This action performs some API calls to GitHub to fetch or close issues and pull requests, set or update labels, add comments, delete branches, etc.

And

> This option aims to limit the number of operations made with the GitHub API to avoid reaching the [rate limit](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting).

If we set it too high we might be blocked due to rate limiting?